cea/demand/demand_main.py

```diff
@@ -69,60 +71,72 @@ def demand_calculation(locator, weather_path, gv, use_dynamic_infiltration_calcu
     t0 = time.clock()
 
     # weather model
-    weather_data = epwreader.epw_reader(weather_path)[['drybulb_C', 'wetbulb_C', 'relhum_percent', 'windspd_ms', 'skytemp_C']]
+    weather_data = epwreader.epw_reader(weather_path)[
+        ['drybulb_C', 'wetbulb_C', 'relhum_percent', 'windspd_ms', 'skytemp_C']]
 
     building_properties, schedules_dict, date = properties_and_schedule(gv, locator)
 
     # in case gv passes a list of specific buildings to simulate.
-    if gv.simulate_building_list:
-        list_building_names = gv.simulate_building_list
-    else:
+    if buildings == []:
         list_building_names = building_properties.list_building_names()
+        print('Running demand calculation for all buildings in the zone')
+    else:
+        list_building_names = gv.simulate_building_list
+        print('Running demand calculation for the next buildings=%s' % list_building_names)
 
     # demand
     if multiprocessing and mp.cpu_count() > 1:
-        thermal_loads_all_buildings_multiprocessing(building_properties, date, gv, locator, list_building_names,
-                                                    schedules_dict, weather_data, use_dynamic_infiltration_calculation)
+        print("Using %i CPU's" % mp.cpu_count())
+        calc_demand_multiprocessing(building_properties, date, gv, locator, list_building_names,
+                                    schedules_dict, weather_data, use_dynamic_infiltration_calculation)
+    else:
+        calc_demand_singleprocessing(building_properties, date, gv, locator, list_building_names, schedules_dict,
+                                     weather_data, use_dynamic_infiltration_calculation)
+
+    # write yearly totals
+    config = cea.config.Configuration()
```

Comment:
this is probably a mistake - the `config` object should be passed in through the `main` function - that allows interfaces (cli, grasshopper, arcgis, tests) to override values in the config object.

Reply:
great, I am passing it around